bitnami/kubectl/1.26/debian-11/Dockerfile

```diff
@@ -39,6 +39,7 @@ RUN mkdir -p /tmp/bitnami/pkg/cache/ ; cd /tmp/bitnami/pkg/cache/ ; \
     done
 RUN apt-get autoremove --purge -y curl && \
     apt-get update && apt-get upgrade -y && \
+    apt-get -y install gettext-base && \
```

Comment:
You can add the required packages to the `install_packages` directive at line 26